### PR TITLE
update hoc.com to link to new hoc.com/learn page

### PR DESCRIPTION
**Pull request by tanyaparker**
image

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/public/activity-guidelines.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ layout: wide
 
 # Lesson guidelines for the Hour of Code™ and Computer Science Education Week
 
-Code.org hosts a variety of Hour of Code™ activities, lessons, tutorials, and videos on the Code.org, Hour of Code, and CSEdWeek website(s). The current list is at [<%= resolve_url('code.org/learn') %>](<%= resolve_url('https://code.org/learn') %>).
+Code.org hosts a variety of Hour of Code™ activities, lessons, tutorials, and videos on the Code.org, Hour of Code, and CSEdWeek website(s). The current list is at [HourOfCode.com/learn](<%= resolve_url('/learn') %>).
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/public/activity-guidelines.md]: do we normally capitalise in HourOfCode.com?  seems slightly odd for a URL

File: pegasus/sites.v3/hourofcode.com/views/whats_next.haml [hunk 2]
```diff
@@ -3,4 +3,4 @@
   %ul{:style=>"line-height:28px"}
     %li=hoc_s(:whats_next_how_to).gsub('/how-to', resolve_url('/how-to'))
     %li=hoc_s(:whats_next_how_to_promote_link).gsub('/promote', resolve_url('/promote'))
-    %li=hoc_s(:whats_next_try).gsub('https://code.org/learn', resolve_url('https://code.org/learn'))
+    %li=hoc_s(:whats_next_try).gsub('%{learn_url}', resolve_url('/learn'))
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/views/whats_next.haml:6]: Presumably this means that on hourofcode.com a translator can put arbitrary HTML into a translation string, and so we have the risk that @Bjvanminnen pointed out in the recent tutorialExplorer PR.  Understand that this is status quo, just wanted to confirm.

**Comment by breville** [comment]
lgtm

File: pegasus/sites.v3/hourofcode.com/views/whats_next.haml (hunk 2 above)
Comment (Bjvanminnen) [reply pegasus/sites.v3/hourofcode.com/views/whats_next.haml:6]: That's a fair point. Let's noticeable bc we don't have the benefits of React forcing us to use dangerouslySetInnerHtml, but I think you're right that these are equally problematic.

File: pegasus/sites.v3/hourofcode.com/public/activity-guidelines.md (hunk 1 above)
Comment (tanyaparker) [reply pegasus/sites.v3/hourofcode.com/public/activity-guidelines.md]: I think it's easier to read and we prefer it for things like #HourOfCode. So I think it helps to reinforce so people don't use #hourofcode (not like it matters _that_ much).

File: pegasus/sites.v3/hourofcode.com/views/whats_next.haml (hunk 2 above)
Comment (tanyaparker) [reply pegasus/sites.v3/hourofcode.com/views/whats_next.haml:6]: That's correct. It reduces the risk slightly, but definitely still a risk.

**Pull request merged**